deeppavlov/models/ner/NER_model.py

```diff
@@ -0,0 +1,286 @@
+# Copyright 2017 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	 http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import string
+from logging import getLogger
+
+import numpy as np
+import tensorflow as tf
+import tensorflow_hub as hub
+from gensim.models import KeyedVectors
+from tensorflow.contrib.layers import xavier_initializer, xavier_initializer_conv2d
+
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.tf_model import LRScheduledTFModel
+
+log = getLogger(__name__)
+
+
+@register('NER_model')
+class NER_model(LRScheduledTFModel):
```

Comment:
Naming should at least hint at how this model is different from the existing `.NerNetwork`

Reply:
changed to `HybridNerModel`